CONTRIBUTING.md

```diff
@@ -0,0 +1,221 @@
+# Contributing guide
+
+We're always looking for new contributors and are happy to help you get started! Some ideas:
+
+- Add a test to a function, testing something that we haven't tested yet.
```

Comment:
The word test repeats itself, and we don't expect trivial tests to be missing (for most functions) so maybe:

```suggestion
- Add a test to a function, like an interesting edge-case we didn't think about.
```